src/server/graphql/subscriptions/integrationLeft.js

```diff
@@ -0,0 +1,26 @@
+import {GraphQLID, GraphQLNonNull} from 'graphql';
+import makeSubscribeIter from 'server/graphql/makeSubscribeIter';
+import {requireSUOrTeamMember} from 'server/utils/authorization';
+import LeaveIntegrationPayload from 'server/graphql/types/LeaveIntegrationPayload';
+import IntegrationService from 'server/graphql/types/IntegrationService';
+
+export default {
+  type: new GraphQLNonNull(LeaveIntegrationPayload),
+  args: {
+    service: {
+      type: new GraphQLNonNull(IntegrationService)
+    },
+    teamId: {
+      type: new GraphQLNonNull(GraphQLID)
+    }
+  },
+  subscribe: (source, {service, teamId}, {authToken, socketId}) => {
+    // AUTH
+    requireSUOrTeamMember(authToken, teamId);
+
+    // RESOLUTION
+    const channelName = `integrationLeft.${teamId}.${service}`;
+    const filterFn = (value) => value.mutatorId !== socketId;
+    return makeSubscribeIter(channelName, filterFn);
```

Comment:
it's all part of the master plan™. 
- Eventually, for scaling purposes, we'll want to move graphql to a microservice. see #1136 
- if we do that, we can't have state in the context, which means we have to do away with both the socket & the exchange in the context (although we could pass in a `socketId`, since that's just a string).
- upon completion, we will be using redis instead of socket cluster. this is gonna yield some good efficiency results & if the day comes when we replace socketcluster with a barebones websocket, that will allow for more natural node child spawning, pm2, debugging, faster webpack for the whole server, etc.

sad as it is, with native subscriptions in graphql, the graphql paradigm obviates the need for socketcluster. Apollo did a nice job of establishing a basic pubsub interface for both redis & rabbitmq so switching between the 2 would literally take 15 minutes. in terms of microservices, it makes much more sense to me to break them into application, socket, and graphql services. for now, we can keep application & socket servers combined. in the future, we could break them apart to maximize RAM so we can store all those AsyncIterators. We can still have an integration microservice, but all it will be doing is listening for webhooks, parsing the payload, and sending it off to the graphql service. 

for now, i'm treating socketcluster like i'm treating cashay. leave the working code in place and incrementally swap it out as we refactor those pieces.